app/src/main/java/ru/yandex/praktikumchatapp/MainActivity.kt

```diff
@@ -71,7 +71,7 @@ fun ChatScreen(
     modifier: Modifier = Modifier
 ) {
     val viewModel = remember { ChatViewModel() }
-    val messagesList = viewModel.messages.observeAsState(emptyList())
+    val messagesList = viewModel.messages.collectAsStateWithLifecycle()
```

Comment:
Здесь можно использовать проперти делегат `by`:
```suggestion
    val messagesList by viewModel.messages.collectAsStateWithLifecycle()
```
Тогда при обращении не будет необходимости везде писать `.value`